public/index.html

```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Account | edX</title>
+      <title>Learner Profile | edX</title>
```

Comment:
Should this be i18n'ed?

Reply:
:O  That's a very good point.  We could dynamically update it once we get the translations...

Reply:
If we do this, the string should include the | edX part, so that the translators can flip it for RTL.

Reply:
If we're going to dynamically update it should we put the user's name or username in there?

Reply:
@abutterworth Possibly - it doesn't do that currently, fwiw.